content-tracker/README.md

```diff
@@ -0,0 +1,55 @@
+# Iroh content tracker
+
+This is an example how to write a simple service using mostly iroh-net.
+The purpose of this service is to track providers of iroh content.
+
+You can *announce* to a tracker that you believe some host has some content.
+The tracker will then periodically *verify* that the content is present.
+Finally, you can *query* a tracker for hosts for a content.
+
```

Comment:
could you add instructions for building? folks might coming here without prior rust knowledge, and will mainly be interested in seeing the example in action.

Something like:
```
## Building from source
Make sure you have an up to date version of [rust]() and [cargo]() installed Then run `cargo build --release` from the root directory. The resulting binary will be in `target/release/tracker`
```

Reply:
OK, will add. I was expecting the target audience to be rust devs, but if we can get more people to use rust this way I am all for it. Just the tooling story for rust is so nice...